Game/ConsoleApplication2/AIManager.cpp

```diff
@@ -0,0 +1,14 @@
+#include "stdafx.h"
+#include "AIManager.h"
+
+//AIManager* AIManager::m_instance = 0;
```

Comment:
Says the person who left a big block of code commented lol. Forgot about this, will fix in morning